templates/zerver/integrations.html

```diff
@@ -1009,6 +1009,49 @@
 
     </div>
 
+    <div id="hellosign" class="integration-instructions">
+
+      <p>
+        Configuring the HelloSign integration is easy, first, create a stream
+        (we recommend naming it <code>hellosign</code>), if no stream is supplied,
+        the default is hellosign if it exists. This does not mean you don't
+        have to establish a stream first.
+      </p>
+
+      <p>
+        Next, go to the Zulip settings page and create a bot named HelloSign.
+        Note the bot API key and add it to the URL below. If you want your
+        notifications to go to a stream other than the default stream
+        of <code>hellosign</code>, modify <code>STREAM_NAME</code> in the below
+        URL to the stream name you want the notifications to go to.
+      </p>
+
+      <p>
+        Go to the account settings page of your HelloSign account and
+        under the Integrations tab, go to API and Account Callback.
+        Add the below URL under the section 'Account Callback', be sure to add the edited URL.
+      </p>
+
+      <p><code>{{ external_api_uri_subdomain }}/v1/external/hellosign?api_key=abcdefgh&amp;stream=STREAM_NAME</code></p>
+
+      <p><img class="screenshot" src="/static/images/integrations/hellosign/000.png" /></p>
+
+      <p>
```

Comment:
I think this paragraph is confusing. When you say that `api_key` must be reconfigured it gives an impression that it changed somewhere. You also should use `STREAM_NAME` instead of `stream=` as you have done above to make this info consistent.